ckanext/canada/templates/internal/package/snippets/resources.html

```diff
@@ -0,0 +1,66 @@
+{% set resources = pkg.resources or [] %}
+{% set can_edit = pkg and h.check_access('package_update', {'id':pkg.id }) %}
+{% if pkg and resources %}
+  {% block resources %}
+    <section class="panel panel-default resources-side-section">
+      {% block resources_inner %}
+        {% block resources_title %}
+          <div class="panel-heading"><i class="fa fa-bars"></i> {{ _("Resources") }}</div>
+        {% endblock %}
+        <div class="panel-body resources-side-panel">
+          {% block resources_list %}
```

Comment:
Can you `{% ckan_extends %}` and only override this block instead?

Reply:
the `resource`s block needs to be overwritten as we are displaying it using panel classes